config/test/test.config.js

```diff
@@ -12,15 +12,27 @@ module.exports = {
     },
     restoreMocks: true,
     resetMocks: true,
-    testEnvironment: "jest-environment-node",
-    testMatch: ["<rootDir>/**/*.test.js"],
+    testEnvironment: "jsdom",
+    testMatch: [
+        "<rootDir>/**/*.test.js",
+        "<rootDir>/**/*_test.js",
```

Comment:
I suspect we don't need this twice?

```suggestion
```

Reply:
These are slightly different.  I'll remove the `*.test.js` for now, but it would be nice to transition to that format since VSCode recognizes `.test.js` files as tests.